server/src/test/java/org/opensearch/index/codec/composite/datacube/startree/StarTreeDocValuesFormatTests.java

```diff
@@ -68,14 +80,40 @@ protected Codec getCodec() {
         final Logger testLogger = LogManager.getLogger(StarTreeDocValuesFormatTests.class);
 
         try {
-            createMapperService(getExpandedMapping("status", "size"));
+            createMapperService(getExpandedMapping());
         } catch (IOException e) {
             throw new RuntimeException(e);
         }
         Codec codec = new Composite99Codec(Lucene99Codec.Mode.BEST_SPEED, mapperService, testLogger);
         return codec;
     }
 
+    private StarTreeMapper.StarTreeFieldType getStarTreeFieldType() {
```

Comment:
We need to write end to end tests with indexwriter and reader since file format is ready.